numba/np/npdatetime.py

```diff
@@ -384,16 +384,12 @@ def impl(context, builder, sig, args):
                 else:
                     builder.store(builder.icmp_unsigned(ll_op, norm_a, norm_b), ret)
             with otherwise:
-                if numpy_support.numpy_version < (1, 16):
-                    # No scaling when comparing NaTs
-                    builder.store(builder.icmp_unsigned(ll_op, va, vb), ret)
+                # NumPy >= 1.16 switched to NaT ==/>=/>/</<= NaT being
```

Comment:
I considered deleting these comments but chose to leave them in as a reminder for anyone revisiting this code as to what the behaviour is around `NaT`... if you think it would be better/more clean to remove them am happy to do so.